drivers/starburst/src/metabase/driver/implementation/execute.clj

```diff
@@ -160,6 +162,14 @@
     (.clearSessionUser (.unwrap conn TrinoConnection))
     nil))
 
+; Metabase tests require a specific error when an invalid number of parameters are passed
+(defn handle-execution-error
+  [e]
+  (log/fatalf (.getMessage e))
+  (if (clojure.string/includes? (.getMessage e) "Incorrect number of parameters")
+    (throw (Exception. "It looks like we got more parameters than we can handle, remember that parameters cannot be used in comments or as identifiers."))
```

Comment:
this is strange, we need to throw a specific error? What happens if we don't?

Reply:
One of the standard Metabase tests fails